index.js

```diff
@@ -1,13 +1,28 @@
 const core = require("@actions/core")
+const github = require('@actions/github')
+const { Octokit } = require("@octokit/rest")
 const { generate } = require("./lib/generate")
 
 async function run() {
   try {
-    const commits = JSON.parse(core.getInput("commits"))
+    const octokit = new Octokit();
+
     const previousReleaseTagNameOrSha = core.getInput("previousReleaseTagNameOrSha")
     const nextReleaseTagName = core.getInput("nextReleaseTagName")
     const nextReleaseName = core.getInput("nextReleaseName")
-    const configPath = core.getInput("configPath")
+    const configPath = core.getInput("configFilePath")
+    
+    let commits
+    const commitsJSON = core.getInput("commits")
+    if (commitsJSON) {
+      commits = JSON.parse(commitsJSON)
+    } else {
+      commits = (await octokit.repos.compareCommits({
```

Comment:
I feel like we should add a test for it. We can mock the call to return a predefined json instead of making a real GH call.

Reply:
Sounds great, will try to do that🙂

Reply:
@mrchief  I have added the test🙂